src/OrchardCore/OrchardCore.Users.Core/Models/LoginSettings.cs

```diff
@@ -4,6 +4,8 @@ namespace OrchardCore.Users.Models;
 
 public class LoginSettings
 {
+    public bool UsersMustValidateEmail { get; set; }
```

Comment:
Why is this better here rather than `RegistrationOptions`? Email validation happens during registration. Even if that registration happens during an external login, it's still a registration.

Reply:
The problem is that RegistrationSettings should only be accessed when the registration feature. We should never try to access it from the main feature. Moving it to the LoginSettings is better because we can access it from the main feature. Also, this setting is check during registration and login so it is not only for registration but for the core feature of the users feature.

Reply:
I made other changes to ensure and undone this change.